### DISCUSSION TITLE
Zooming in and out over the duration of the video

### DISCUSSION THREAD

**Discussion by MDrooker** [remotion-dev/remotion#639] [Q&A] [answered]
Hey All-
Looking for a better way todo something-
I have a background image that i am doing an interpolate on to zoom in over some time period.  Looks something like-
``` 
const zoom =
        interpolate(frame,
            [0, durationInFrames / 4], [1, 1.2],
            { extrapolateRight: 'clamp' });
```

So for this example I am zooming in for 1/4 of the video.  For the next 1/4 I want to zoom out...then zoom back in...then zoom back out.

This will create a nice movement behind to presentation.

I know I can create 4 zoom interpolate functions...and just hard code the ```     transform: `scale(${zoom})`,```
But is there a cleaner way todo this?  Kinda new to the interpolate function world, and bet Im totally missing something to make these KenBurns effects a ton cleaner.

Thanks much-
MJD

**Comment by Iamshankhadeep** [comment, marked as answer]
@MDrooker we do support array on interpolate so you can try this.
```const zoom =
        interpolate(frame,
            [0, durationInFrames / 4, 2*(durationInFrames / 4), 3 * (durationInFrames / 4), durationInFrames ], [1, 1.2, 1, 1.2, 1],
            { extrapolateRight: 'clamp' });
```
### Note:
This is a Remotion2.0 feature
[Doc](https://remotion.dev/docs/interpolate)
let me know if it helps.

**Reply by MDrooker** [reply]
Totally helps..
And I can use that for sure
